src/components/ImportForm/utils/submit-utils.ts

```diff
@@ -68,8 +70,16 @@ export const createResources = async (formValues: ImportFormValues, strategy: Im
     componentAnnotations = { [SAMPLE_ANNOTATION]: 'true' };
   }
 
+  const integrationTestValues: IntegrationTestFormValues = {
+    name: `${applicationName}-enterprise-contract`,
+    pipeline: 'enterprise-contract',
+    bundle: 'quay.io/redhat-appstudio-tekton-catalog/pipeline-enterprise-contract:devel',
+    optional: true,
```

Comment:
Can we get a spec.params in here I wonder? No idea if this might work, but...

```suggestion
    optional: true,
    spec: {
      params: [
        { 
          name: 'POLICY_CONFIGURATION',
          value: 'github.com/enterprise-contract/config//slsa3',
        },  
      ]
    }
```

Reply:
The UI support is not quite there yet for this. I think we should do it as follow up?

Reply:
Yeah... So there's also UI to edit an existing IntegrationTestScenario which doesn't support params yet. We should wait for that. In the mean time, I'll update the default ECP.